pytorch_toolkit/object_detection/model_templates/face-detection/face-detection-0202/template.yaml

```diff
@@ -43,7 +43,10 @@ output_format:
   openvino:
     default: true
     input_format: BGR
-compression: null
+optimisations:
+  nncf_quantization:
+    config: "compression_config.json"
```

Comment:
Could you please remove string enclosing?

Reply:
Fixed